package.json

```diff
@@ -334,20 +335,25 @@
                     "when": "view == extension.vsKubernetesExplorer && viewItem == vsKubernetes.resource.pod"
                 },
                 {
-                    "command": "extension.vsKubernetesTerminal",
+                    "command": "extension.vsKubernetesDeleteNow",
```

Comment:
I'm a bit wary of having both `Delete` and `Delete Now` in the context menu.  It feels like we're starting to overload the menu with possible options for one particular command.  I have a couple of alternative suggestions:

1. Have `Delete Now` in the palette only, so it takes more effort to select as befits a more destructive command.

2. As we already display a confirmation for the `Delete` command, add a "now" option to that confirmation.  E.g. 'pod/foo-123456 will be deleted. [Delete Gracefully] [Delete Now] [Cancel]'.  But perhaps that makes it too easy to click Delete Now when you meant Delete Gracefully, I'm not sure.

Reply:
I'm fine with `2.)`, however, that overloaded confirmation window will now be surprisingly different to unsuspecting users, which may or may not create even greater confusion. I'd say keep it simple(r)?